src/pages/home/report/ReportActionsView.js

```diff
@@ -185,6 +177,13 @@ class ReportActionsView extends React.Component {
             }
         }
 
+        if (this.props.isWindowVisible !== prevProps.isWindowVisible && isReportFullyVisible) {
+            // If the app user becomes active and they have no unread actions we clear the new marker to sync their device
+            // e.g. they could have read these messages on another device and only just become active here
+            Report.openReport(this.props.report.reportID);
+            this.setState({newMarkerSequenceNumber: 0});
+        }
```

Comment:
This works, but I _think_ I prefer the approach of having a listener in `componentDidMount()` - and it would also require less changes. There's a slight downside to this new approach which is that we will be re-rendering this component when the app becomes visible and then setting state again.